toxcore/tox.h

```diff
@@ -2933,6 +2933,167 @@ uint16_t tox_self_get_udp_port(const Tox *tox, TOX_ERR_GET_PORT *error);
+                                                TOX_ERR_CONFERENCE_PEER_QUERY *error);
+    bool (*tox_conference_peer_get_name)(const Tox *tox, uint32_t conference_number, uint32_t peer_number, uint8_t *name,
+                                         TOX_ERR_CONFERENCE_PEER_QUERY *error);
+    bool (*tox_conference_peer_get_public_key)(const Tox *tox, uint32_t conference_number, uint32_t peer_number,
+                                               uint8_t *public_key, TOX_ERR_CONFERENCE_PEER_QUERY *error);
+    bool (*tox_conference_peer_number_is_ours)(const Tox *tox, uint32_t conference_number, uint32_t peer_number,
+                                               TOX_ERR_CONFERENCE_PEER_QUERY *error);
+    bool (*tox_conference_invite)(Tox *tox, uint32_t friend_number, uint32_t conference_number,
+                                  TOX_ERR_CONFERENCE_INVITE *error);
+    uint32_t (*tox_conference_join)(Tox *tox, uint32_t friend_number, const uint8_t *cookie, size_t length,
+                                    TOX_ERR_CONFERENCE_JOIN *error);
+    bool (*tox_conference_send_message)(Tox *tox, uint32_t conference_number, TOX_MESSAGE_TYPE type, const uint8_t *message,
+                                        size_t length, TOX_ERR_CONFERENCE_SEND_MESSAGE *error);
+    size_t (*tox_conference_get_title_size)(const Tox *tox, uint32_t conference_number, TOX_ERR_CONFERENCE_TITLE *error);
+    bool (*tox_conference_get_title)(const Tox *tox, uint32_t conference_number, uint8_t *title,
+                                     TOX_ERR_CONFERENCE_TITLE *error);
+    bool (*tox_conference_set_title)(Tox *tox, uint32_t conference_number, const uint8_t *title, size_t length,
+                                     TOX_ERR_CONFERENCE_TITLE *error);
+    size_t (*tox_conference_get_chatlist_size)(const Tox *tox);
+    void (*tox_conference_get_chatlist)(const Tox *tox, uint32_t *chatlist);
+    TOX_CONFERENCE_TYPE (*tox_conference_get_type)(const Tox *tox, uint32_t conference_number,
+                                                   TOX_ERR_CONFERENCE_GET_TYPE *error);
+    bool (*tox_friend_send_lossy_packet)(Tox *tox, uint32_t friend_number, const uint8_t *data, size_t length,
+                                         TOX_ERR_FRIEND_CUSTOM_PACKET *error);
+    bool (*tox_friend_send_lossless_packet)(Tox *tox, uint32_t friend_number, const uint8_t *data, size_t length,
+                                            TOX_ERR_FRIEND_CUSTOM_PACKET *error);
+    void (*tox_callback_friend_lossy_packet)(Tox *tox, tox_friend_lossy_packet_cb *callback);
+    void (*tox_callback_friend_lossless_packet)(Tox *tox, tox_friend_lossless_packet_cb *callback);
+    void (*tox_self_get_dht_id)(const Tox *tox, uint8_t *dht_id);
+    uint16_t (*tox_self_get_udp_port)(const Tox *tox, TOX_ERR_GET_PORT *error);
+    uint16_t (*tox_self_get_tcp_port)(const Tox *tox, TOX_ERR_GET_PORT *error);
+} ToxApi;
+
+extern ToxApi tox_api;
```

Comment:
No, users are supposed to copy it to create a flavor of the API based on the original API.